src/meson.build

```diff
@@ -0,0 +1,241 @@
+config_cfg = configuration_data()
+config_cfg.set('VERSION', intel_vaapi_driver_version)
+config_cfg.set('VA_DRIVER_INIT_FUNC', va_driver_init_func)
+config_cfg.set('INTEL_DRIVER_MAJOR_VERSION', intel_vaapi_driver_major_version)
+config_cfg.set('INTEL_DRIVER_MINOR_VERSION', intel_vaapi_driver_minor_version)
+config_cfg.set('INTEL_DRIVER_MICRO_VERSION', intel_vaapi_driver_micro_version)
+config_cfg.set('INTEL_DRIVER_PRE_VERSION', intel_vaapi_driver_pre_version)
+config_cfg.set10('HAVE_HYBRID_CODEC', get_option('enable_hybrid_codec'))
+config_cfg.set10('HAVE_VA_X11', not get_option('disable_x11'))
```

Comment:
Need to check for libva-x11... otherwise compilation fails if libva-x11 is missing.  Better to fail during configure.